platooning_control/config/parameters.yaml

```diff
@@ -1,19 +1,19 @@
-standStillHeadway: 12.0 # Standstill gap between vehicles (m)
-maxAccel: 1.5 # Maximum acceleration absolute value used in controller filters (m/s^2)
-Kp: 0.4 # Proportional weight for PID controller
-Kd: 0.0  # Derivative Weight for PID controller  
-Ki: 0.03 # Integral weight for PID controller
-maxValue: 2.0 # Max value to restrict speed adjustment at one time step (limit on delta_v) (m/s)
-minValue: -10.0 # Min value to restrict speed adjustment at one time step (limit on delta_v) (m/s)
-cmdTmestamp: 100 # Timestep to calculate ctrl commands (ms)
-adjustmentCap: 15.0 # Adjustment cap for speed command (m/s)
+stand_still_headway: 12.0 # Standstill gap between vehicles (m)
```

Comment:
lets include the units in the names. because in general it is a good practice and there are differences in units in this package itself (ms vs s). So like:
`stand_still_headway_meter` or `max_accel_ms2` etc.
This applies to all instances of the variables being used in the cpp code itself as well.